gradle.properties

```diff
@@ -32,4 +32,4 @@ org.gradle.parallel=true
 org.gradle.configureondemand=true
 # explicitly disable the configuration cache
 org.gradle.configuration-cache=false
-org.gradle.jvmargs=-Xmx1024m
+org.gradle.jvmargs=-Xmx2048m
```

Comment:
Java CI failed with "Java heap space" error.

Reply:
Did it just fail here? What's changed?

Reply:
It could be an intermittent issue. After one of my pushes, the CI that ran failed. That is the reason I made this change, to get it to pass. I can try reverting this gradle change and see if the CI passes. If it does, that means it is an intermittent problem.

Reply:
I reverted the gradle change and the CI passed. It appears to be an inconsistent issue.